one-ui/src/test/java/com/marklogic/hub/curation/controllers/EnvironmentControllerTest.java

```diff
@@ -49,10 +53,8 @@
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.web.multipart.MultipartFile;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
we had better use single class import. You can check your java code style configuration. Make sure the option "Use single class import" checked.

Reply:
I have single class import checked already:

![image](https://user-images.githubusercontent.com/16124406/75054789-99275a80-5488-11ea-8b45-5c06e6cb2b9b.png)

Reply:
<img width="769" alt="Screen Shot 2020-02-21 at 9 31 04 AM" src="https://user-images.githubusercontent.com/444927/75057028-f32a1f00-548c-11ea-9637-4fbca425412b.png">